CONTRIBUTING.md

```diff
@@ -67,22 +67,24 @@ $ git push -u <your username> <the name of your branch>
 are adding code which would be untested, please consider adding tests to cover
 that code.
 
+- For new features or changes users should be aware of, add an entry to `CHANGELOG.md`.
```

Comment:
should we also add a reminder in a PR template?

Reply:
realized that we currently have no PR templates...if we did, it'd be nice to add a reminder, but..